doc/classes/Callable.xml

```diff
@@ -13,21 +13,21 @@
 		    prints(arg1, arg2, arg3)
 
 		func test():
-		    callable.call("hello", "world")  # Prints "hello world".
+		    callable.call("hello", "world")  # Prints "hello world ".
 		    callable.call(Vector2.UP, 42, callable)  # Prints "(0, -1) 42 Node(Node.gd)::print_args".
 		    callable.call("invalid")  # Invalid call, should have at least 2 arguments.
 		[/gdscript]
 		[csharp]
-		Callable callable = new Callable(this, nameof("printArgs"));
+		Callable callable = new Callable(this, nameof(PrintArgs));
 		public void PrintArgs(object arg1, object arg2, object arg3 = "")
 		{
 		    GD.PrintS(arg1, arg2, arg3);
 		}
 
 		public void Test()
 		{
-		    callable.Call("hello", "world"); // Prints "hello world".
-		    callable.Call(Vector2.Up, 42, callable); // Prints "(0, -1) 42 Node(Node.gd)::print_args".
+		    callable.Call("hello", "world"); // Prints "hello world ".
+		    callable.Call(Vector2.Up, 42, callable); // Prints "(0, -1) 42 Node(Node.cs)::PrintArgs".
```

Comment:
`Node(Node.cs)::PrintArgs` part is actually my guess since I can't test it. Please someone confirm/correct it.

Reply:
It seems correct, but I have to double check it this evening

Reply:
The output is actually `Godot.Callable`, because the Callable is not (usable) implemented in C#. I think the output should be as you wrote, but for the time being there is no output. (There is no call function) and yes I am to blame. I wrote that code translation and I didn't test this particular script :sweat_smile: